tests/e2e/cucumber/environment/index.ts

```diff
@@ -231,10 +230,10 @@ const setAdminToken = async (user: User) => {
   return await setAccessToken(user.id)
 }
 
-const setAdminTokenFromLogin = async (browser: Browser) => {
-  return await getTokenFromLogin({ browser })
-}
-
 const setKeycloakAdminTokenfromApi = async (user: User) => {
   return await getAccessToken(user)
 }
+
+const setKeycloakAdminTokenForoCIS = async (user: User) => {
+  return await setKeyCloakAccessToken(user.id)
+}
```

Comment:
I do not see the need for separate functions. You can use `getAccessToken` and `setKeyCloakAccessToken` directly

Reply:
and you should rename `getAccessToken` to somthing that states that it is for keycloak server